apps/meteor/app/livechat/server/api/v1/room.ts

```diff
@@ -352,7 +352,12 @@ API.v1.addRoute(
 
 API.v1.addRoute(
 	'livechat/room.visitor',
-	{ authRequired: true, permissionsRequired: ['view-l-room'], validateParams: isPUTLivechatRoomVisitorParams, deprecationVersion: '7.0.0' },
+	{
+		authRequired: true,
+		permissionsRequired: ['change-livechat-room-visitor'],
```

Comment:
```suggestion
		permissionsRequired: ['change-livechat-room-visitor', 'view-l-room'],
```

Reply:
Considering previously we needed both these permissions to change room's visitor

Reply:
view-l-room was there, but was actually not of any use. Inside the only function call, it checked for `change-livechat-room-visitor` which means l-room alone was useless.

So i removed it :)

Reply:
Yeah makes sense :+1: Just thought about raising this since this is a refactoring PR which is changing some behaviour of the product

Reply:
Thank you! Yeah, the thing is, the behavior keeps unchanged. We just check the permission before